apps/finance/test/finance.js

```diff
@@ -255,17 +355,6 @@ contract('Finance App', accounts => {
         assert.equal(await finance.currentPeriodId(), 1, 'should have transitioned 1 periods')
     })
 
-    it("escapes hatch, recovers ETH", async () => {
```

Comment:
Removed this test as it is testing the same as the `records ETH deposits using fallback` test case